helix-stdx/src/env.rs

```diff
@@ -8,6 +8,15 @@ use std::{
 use once_cell::sync::Lazy;
 
 static CWD: RwLock<Option<PathBuf>> = RwLock::new(None);
+// previous working directory, stored to allow `:cd -`
+static PREV_CWD: RwLock<Option<PathBuf>> = RwLock::new(None);
```

Comment:
We keep the `CWD` as a static so that we can access it in places where we don't have access to the `Editor`. For this feature though we only need to access the previous cwd within the `:cd` command (where we have an exclusive borrow of the Editor) so we can store a `previous_cwd: Option<PathBuf>` on the `Editor` rather than dealing with all of the static and locking business here

Reply:
Ok, I got it. I updated the implementation which now stores a new `last_cwd` on the `Editor`

Reply:
Before the `Editor` is even initialized I'm just passing `&mut None` to the `set_current_working_dir`